### PR TITLE
Reworking the logging system

### PR DESCRIPTION
**Pull request by PancakeTAS**
so much duplicate and unnecessary data

### CODE REVIEW

File: Gameserver/src/main/java/com/minecrafttas/tasbattle/lobby/LobbyTimer.java [hunk 1]
```diff
@@ -22,14 +22,16 @@
...
-	private BukkitTask task;
-	private int startTime, time;
-	private int minPlayers, maxPlayers;
-	private Consumer<List<Player>> start;
+	private final BukkitTask task;
+	private final int startTime;
+    private int time;
+	private final int minPlayers;
+    private final int maxPlayers;
+	private final Consumer<List<Player>> start;
```
Comment (ScribbleTAS) [inline Gameserver/src/main/java/com/minecrafttas/tasbattle/lobby/LobbyTimer.java:34]: Some weird indentation issues here

File: Proxy/src/main/java/com/minecrafttas/tasbattle/TASBattleProxy.java [hunk 2]
```diff
@@ -22,35 +22,37 @@
...
 	private ChatSystem chatSystem;
 	private CustomTabList customTabList;
-	private ProxyTelemetry proxyTelemetry;
 
 	/**
      * Construct proxy plugin
      * @param server Proxy server instance
+	 * @param logger Logger instance
+	 * @param dataDirectory Data directory
      */
```
Comment (ScribbleTAS) [inline Proxy/src/main/java/com/minecrafttas/tasbattle/TASBattleProxy.java:37]: Another set of indentation issues

**Review by ScribbleTAS — APPROVED** [review]
Tell me if I should ignore indentation stuff in the future

**Comment by PancakeTAS** [comment]
urgh, the joy of switching between eclipse and intellij, windows and linux.
just ignore it for now, if I ever come back to this I'll remember to fix the indentation for all files (in it's own pr probably)

**Review by ScribbleTAS — APPROVED** [review]

**Pull request merged**